translations/monero-core_es.ts

```diff
@@ -610,27 +610,27 @@
     <message>
         <location filename="../pages/Mining.qml" line="148"/>
         <source>Start mining</source>
-        <translation>Empezar minado</translation>
+        <translation>Empezar a minar</translation>
     </message>
     <message>
         <location filename="../pages/Mining.qml" line="158"/>
         <source>Error starting mining</source>
-        <translation>Error arrancando el minado</translation>
+        <translation>Error arrancando el proceso de minado</translation>
     </message>
     <message>
         <location filename="../pages/Mining.qml" line="159"/>
         <source>Couldn&apos;t start mining.&lt;br&gt;</source>
-        <translation>No se pudo empezar el minado.&lt;br&gt;</translation>
+        <translation>No se pudo empezar el proceso de minado.&lt;br&gt;</translation>
     </message>
     <message>
         <location filename="../pages/Mining.qml" line="161"/>
         <source>Mining is only available on local daemons. Run a local daemon to be able to mine.&lt;br&gt;</source>
-        <translation>El minado sólo está disponible en nodos locales. Arranque un daemon local para poder minar.&lt;br&gt;</translation>
+        <translation>El minado solo está disponible en nodos locales. Arranque un daemon local para poder empezar.&lt;br&gt;</translation>
```

Comment:
La minería.... .. para poder minar.

Mira lo que encontré en la terminología windows:
modelo de minería de datos
An object that contains the definition of a data mining process and the results of the training activity. For example, a data mining model may specify the input, output, algorithm, and other properties of the process and hold the information gathered during the training activity, such as a decision tree.
*Si miras lo que hace el deamon, es una ejecución de actividades que se desprenden de ramas de un arbol (mapa conceptual)